sdk/elixir/lib/dagger/engine_conn.ex

```diff
@@ -5,7 +5,7 @@ defmodule Dagger.EngineConn do
 
   defstruct [:port, :token, :session_pid]
 
-  @dagger_cli_version "0.6.2"
+  @dagger_cli_version "0.6.4"
```

Comment:
Was this meant to be `0.8.0`?

BTW, `0.8.1` is coming in hot, most likely in the next few hours: https://github.com/dagger/dagger/milestone/19

Reply:
I’ll re-bump to 0.8.1. 🎉

Reply:
Fix in this issue https://github.com/dagger/dagger/pull/5565/commits/7523310d8cb6a777aace0a283e1d1491a09f1942